force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -2810,7 +2810,15 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Error when Number of Days for Lapsed is smaller than Number of Days For Closed</shortDescription>
-        <value>Number of Days for Closed must be higher than the Number of Days for Lapsed</value>
+        <value>Number of Days for Closed must be higher than the Number of Days for Lapsed.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_StatusAutomationNegativeNumberOfDays</fullName>
```

Comment:
@sam-knox Adding one more custom label for the following -> Number of Days for Lapsed/Closed is negative.

Reply:
@andrewyu-salesforce - Ok. Label text looks good to me.